### ISSUE TITLE
Imported NuGet .targets causes additional project platforms to be displayed

### ISSUE THREAD

**Issue by sgarske** [MicrosoftEdge/WebView2Feedback#1755]
**Description**

After installing the `Microsoft.Web.Webview2` NuGet package onto a C# class library (.NET Framework) project, Visual Studio will display additional project platforms that are not defined in the `.csproj`.

Default project platforms after creating project
![image](https://user-images.githubusercontent.com/7501210/134237062-c2695d18-ca92-4958-a829-a2d5d20c5dd4.png)

After installing NuGet package
![image](https://user-images.githubusercontent.com/7501210/134237080-0b723be3-1143-43a9-b41e-bca2bca9cabb.png)

**Version**
SDK: At least `1.0.705.50` of `Microsoft.Web.Webview2` NuGet package or later (also confirmed with `1.0.1018-prerelease` )
Runtime: N/A?
Framework: Confirmed with a WPF application project/solution and a C# Wix Custom Action library
OS: Win10

**Repro Steps**

1. Create a new C# Class Library (.NET Framework) project
2. Install latest `Microsoft.Web.Webview2` NuGet package
3. Close/re-open solution
4. View the detected "Platforms" in the project properties 

I expected no additional Platforms to show besides what are defined by my project.  By installing the package and the import of the `.targets`, additional platforms show.

**Additional context**

I believe this has to do with the setting of `EffectivePlatform` at the beginning of `Common.targets` ( automatically imported by `Microsoft.Web.WebView2.targets` )
```
<?xml version="1.0"?>
<Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
  <!-- The Platform property is Win32 or x64 for C++ projects. Convert from Win32 to x86 to match our directory structure.
    If PlatformTarget property is set, then use that to determine EffectivePlatform for .NET projects.
    See conditions: https://docs.microsoft.com/en-us/visualstudio/msbuild/msbuild-conditions?view=vs-2019
    and PropertyGroup: https://docs.microsoft.com/en-us/visualstudio/msbuild/propertygroup-element-msbuild?view=vs-2019
  -->
  <PropertyGroup>
    <EffectivePlatform>$(Platform)</EffectivePlatform>
    <EffectivePlatform Condition="'$(Platform)' == 'Win32'">x86</EffectivePlatform>
    <EffectivePlatform Condition="'$(Platform)' == 'Any CPU'">x86</EffectivePlatform>
    <EffectivePlatform Condition="'$(Platform)' == 'AnyCPU'">x86</EffectivePlatform>
...
```

I believe by just checking for conditional platforms that don't exist in the project, it triggers the project UI to show them as possible options.  This causes confusion of which platforms are actually available.

I removed these lines and the issue went away.

Honestly, it feels like it may be an issue with Visual Studio and the project structure, but maybe that's just how it works? Possibly the NuGet targets can add a conditional to the whole `PropertyGroup` since it's not even needed in the .NET project case.

Thanks for reading/considering.

[AB#36410393](https://microsoft.visualstudio.com/90b2a23c-cab8-4e7c-90e7-a977f32c1f5d/_workitems/edit/36410393)

**Comment by champnic** [comment]
Thanks for the bug report @sgarske! I've added it to our backlog. It is surprising that Visual Studio would be so proactive about adding platforms based on a condition check!

**Comment by champnic** [comment]
I've just put in a quick fix for this. I think it just missed the cutoff for the next prerelease SDK, but should be available in the following one.

**Comment by champnic** [comment]
Available in SDKs version [1.0.2535.41](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.2535.41)+. Thanks!
